src/pages/posts/page/[page].js

```diff
@@ -0,0 +1,35 @@
+import { getAllPosts, getPagesCount, getPaginatedPosts } from 'lib/posts';
+
+import TemplateArchive from 'templates/archive';
+
+export default function Posts({ posts, pagination }) {
+  const title = 'All Posts';
+  const slug = 'posts';
+
+  pagination.basePath = '/posts';
+
+  return <TemplateArchive title={title} posts={posts} slug={slug} pagination={pagination} />;
+}
+
+export async function getStaticProps({ params = {} } = {}) {
+  const { posts, pagination } = await getPaginatedPosts(params?.page);
+  return {
+    props: {
+      posts,
+      pagination,
+    },
+  };
+}
+
+export async function getStaticPaths() {
+  const { posts } = await getAllPosts();
+  const pagesCount = getPagesCount(posts);
+  let paths = [];
```

Comment:
did you know you could do something like:

```
[...new Array(pagesCount)].map((nothing, i) => {
  return { params: { page: String(i + 1) } }
});
```

not saying to do it but just thought id JSYK :)

Reply:
Nicee😎! I liked, will change it. thanks!